src/App.tsx

```diff
@@ -1,61 +1,90 @@
+import React, { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoType } from './types/type';
 
 export const App = () => {
+  const [todos, setTodos] = useState<TodoType[]>(todosFromServer);
+
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [hasTitleError, setTitleError] = useState(false);
+  const [hasUserError, setUserError] = useState(false);
+
+  const handleTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```
Pay attention to naming